test/npu-xrt/runtime_cumsum/aie2.mlir

```diff
@@ -0,0 +1,89 @@
+module {
+    aie.device(npu1_1col) {
+
+        func.func private @sum(memref<16xi32>, memref<16xi32>)
+        func.func private @zero(memref<16xi32>)
+
+        %shim_noc_tile_0_0 = aie.tile(0, 0)
+        %mem_tile_0_1 = aie.tile(0, 1)
+        %tile_0_2 = aie.tile(0, 2)
+
+        aie.objectfifo @mem_In(%shim_noc_tile_0_0, {%mem_tile_0_1}, 1 : i32) : !aie.objectfifo<memref<16xi32>> 
+        aie.objectfifo @act_In(%mem_tile_0_1, {%tile_0_2}, 1 : i32) : !aie.objectfifo<memref<16xi32>> 
+        aie.objectfifo.link [@mem_In] -> [@act_In]([] [])
+        aie.objectfifo @out(%tile_0_2, {%mem_tile_0_1}, 1 : i32) : !aie.objectfifo<memref<16xi32>> 
+        aie.objectfifo @mem_out(%mem_tile_0_1, {%shim_noc_tile_0_0}, 1 : i32) : !aie.objectfifo<memref<16xi32>> 
+        aie.objectfifo.link [@out] -> [@mem_out]([] [])
+
+        %rtp2 = aie.buffer(%tile_0_2) {sym_name = "rtp2"} : memref<16xi32> 
+
+        %core_0_2 = aie.core(%tile_0_2) {
+            %c0 = arith.constant 0 : index
+            %cmax = arith.constant 294967295 : index
+            %c1 = arith.constant 1 : index
+            %c2 = arith.constant 2 : index
+            %c4 = arith.constant 4 : index
+
+            scf.for %arg0 = %c0 to %cmax step %c1 {
+                %Out = aie.objectfifo.acquire @out(Produce, 1) : !aie.objectfifosubview<memref<16xi32>>
+                %elemout = aie.objectfifo.subview.access %Out[0] : !aie.objectfifosubview<memref<16xi32>> -> memref<16xi32>
+
+                func.call @zero(%elemout) : (memref<16xi32>) -> ()
+
+                scf.while (%arg1 = %c1) : (index) -> (index) {
+
+                    %In = aie.objectfifo.acquire @act_In(Consume, 1) : !aie.objectfifosubview<memref<16xi32>>
+                    %elemin = aie.objectfifo.subview.access %In[0] : !aie.objectfifosubview<memref<16xi32>> -> memref<16xi32>
+
+                    func.call @sum(%elemin, %elemout) : (memref<16xi32>, memref<16xi32>) -> ()
+
+                    aie.objectfifo.release @act_In(Consume, 1)
+
+                    %loopnumber = memref.load %rtp2[%arg0] : memref<16xi32>
+                    %loopnumber2index = arith.index_cast %loopnumber : i32 to index
+
+                    %next = arith.addi %arg1, %c1 : index
+                    %cond = arith.cmpi slt, %arg1, %loopnumber2index : index
+                    scf.condition(%cond) %next : index
+                } do {
+                ^bb0(%arg2: index):
+                    scf.yield %arg2: index
+                }
+                aie.objectfifo.release @out(Produce, 1)
+            }
+            aie.end
+        } {link_with = "sum.o"}
+
+        aiex.runtime_sequence @sequence(%xy: memref<128xi32>) {
+            aiex.npu.rtp_write(@rtp2, 0, 1)
+            aiex.npu.dma_memcpy_nd(0, 0, %xy[0, 0, 0, 0][1, 1, 1, 16][0, 0, 0, 1]) {id = 0 : i64, metadata = @mem_In} : memref<128xi32>
```

Comment:
Recently in mlir-aie there has been a change in the `aiex.npu.dma_memcpy_nd` op that removed the x and y arguments. PR: https://github.com/Xilinx/mlir-aie/commit/d9c362a438210d8b7d77440f8236d2282790f208 Could you please remove the x and y arguments and try again?